default.json

```diff
@@ -1,4 +1,46 @@
 {
+  "packageRules": [
+    {
+      "matchPackageNames": [
+        "golang/go",
+        "golang/tools",
+        "kubernetes/kubectl"
+      ],
+      "matchPaths": [
```

Comment:
would this still match a nested `aqua.yaml`, like `foo/bar/aqua.yaml`, currently the existing regex `"fileMatch": ["\\.?aqua\\.ya?ml"],` matches those, I would assume the `matchPaths` is not needed then?

Reply:
> would this still match a nested aqua.yaml, like foo/bar/aqua.yaml

Yes. https://docs.renovatebot.com/configuration-options/#matchpaths

This is required to prevent this preset from affecting packages that aren't managed by aqua.

Reply:
awesome, this is great. I guess this changes also needs to go into `file.json`

Reply:
I've updated file.json too.
https://github.com/aquaproj/aqua-renovate-config/pull/207/commits/453144b1837b4ad06424189993452360fefbe4c4